### ISSUE TITLE
Cannot find module 'types' or its corresponding type declarations in node_modules/posthog-js/dist/module.d.ts:1:52

### ISSUE THREAD

**Issue by jeremytenjo** [PostHog/posthog#16188]
## Bug description

*Please describe.*  
*If this affects the front-end, screenshots would be of great help.*  
![image](https://github.com/PostHog/posthog/assets/25231946/2d02b493-9203-47a9-be49-880e5a762c1d)

```js
node_modules/posthog-js/dist/module.d.ts:1:52 - error TS2307: Cannot find module 'types' or its corresponding type declarations.

1 import { SurveyCallback as SurveyCallback$1 } from 'types';
```

*If you are on PostHog Cloud it would be really valuable if you can share any links where the problem occurs. This speeds up our ability to troubleshoot tremendously.* 

## How to reproduce

1. install "posthog-js": "1.68.2"
2. npm run tsc --noemit

## Environment

```json 
"posthog-js": "1.68.2",
```

Thank you :)

**Comment by neilkakkar** [comment]
(cc: @liyiy )

**Comment by liyiy** [comment]
Hey there, how were you running into this error? Did you try deleting node modules and reinstalling?

**Comment by jkonst** [comment]
I can confirm this error on older versions as well e.g. `1.57.2` and typescript `4.8.4`. This error does not appear only if you have `"skipLibCheck": true` in your tsconfig.json.

Actually this issue should be transferred to [posthog-js issues](https://github.com/PostHog/posthog-js/issues). It has nothing to do with this codebase.

**Comment by jeremytenjo** [comment]
@jkonst Right, I posted it there https://github.com/PostHog/posthog-js/issues/712

Thanks.
